resources/user-agents/apps/microsoft/bing-search-app-android-6.json

```diff
@@ -0,0 +1,184 @@
+                    "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Chrome/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "devices": {
+                        "GT-I9195": "Samsung GT-I9195",
+                        "LENNY": "Wiko Lenny",
+                        "Lenovo A616": "Lenovo A616",
+                        "SM-G850F*": "Samsung SM-G850F",
+                        "Tab2A7-10*": "Lenovo Tab 2 A7-10"
+                    },
+                    "platforms": [
+                        "Android_4_4"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Chrome/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "platforms": [
+                        "Android_7_0",
+                        "Android_6_0",
+                        "Android_5_1", "Android_5_0",
+                        "Android_4_4",
+                        "Android"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "engine": "WebKit",
+                    "devices": {
+                        "C1905": "Sony C1905",
+                        "GT-I9300": "Samsung GT-I9300"
+                    },
+                    "platforms": [
+                        "Android_4_3"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "engine": "WebKit",
+                    "devices": {
+                        "ASUS Transformer Pad TF300TL": "Asus TF300TL",
+                        "GT-I9195": "Samsung GT-I9195",
+                        "GT-P5113": "Samsung GT-P5113"
+                    },
+                    "platforms": [
+                        "Android_4_2"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "engine": "WebKit",
+                    "devices": {
+                        "GT-I8190": "Samsung GT-I8190",
+                        "LG-E435f": "LG E435f"
+                    },
+                    "platforms": [
+                        "Android_4_1"
+                    ]
+                },
+                {
+                    "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Version/* Mobile Safari/* BingWeb/#MAJORVER#.#MINORVER#*",
+                    "engine": "WebKit",
+                    "platforms": [
+                        "Android_4_3", "Android_4_2",
```

Comment:
Same here, there's an `Android_4_1` platform above, should add here too.

Reply:
changed